index.src.html

```diff
@@ -752,13 +757,30 @@ <h3 id="set-referrer-policy">
     </li>
   </ol>
 
+  <h3 id="set-requests-referrer-policy-on-redirect">
+    Set |request|'s referrer policy on redirect
+  </h3>
+
+  Given a {{Request}} |request| and a {{Response}} |actualResponse|,
```

Comment:
Done, I think (did I do that right?). Also, presumably this should apply to all the other instances of this in various algorithms also? If so I'll do that separately.